content/blog/overview-of-the-platform-level-data-model-for-redfish®-device-enablement-standard.md

```diff
@@ -0,0 +1,158 @@
+
+### Network adapters
+
+Several network adapter suppliers implement PLDM for RDE in their devices. The [Intel E810 XXVDA2 Network Adapter](https://www.hpe.com/psnow/doc/a00073559enw.html?jumpid=in_pdp-psnow-qs) support this new technology in its firmware version 3.10. This firmware is packaged in a file called `HPE_E810_XXVDA2_SD_3p10_PLDMoMCTP_8000AD4A.fwpkg` mentioning explicitly PLDM over MCTP, meaning that PLDM messages between the Management Controller and the device are transported by the Management Component Transport Protocol mentioned above.
+
+When this firmware is deployed a network adapter containing string `/DE` is present under the `NetworkAdapter.` data type location. The next picture extracts three properties (URI, Name and Settings URI) from the `NetworkAdapter.` data type of a system containing two network adapters: an HPE 631FLR-SFP28 and an Intel E810. The HPE 631FLR-SFP28 adapter does not support RDE and is referred as a `/DC` device. Moreover, it contains a settings URI used to store modifications before being transferred to the device during the next system reboot.
+
+The Intel E810 adapter is referred as a `/DE` device supporting RDE. Moreover, it does not contain any settings URI because modifications are transferred to the device in real time by the iLO and no server reboot is required to take those modification into account.
+
+![Setting an RDE network adapter property with iLOrest](/img/networkadapters.png "Setting an RDE network adapter property with iLOrest")
+
+The following picture shows a real-time property modification on port 0 of an Intel E810 network adapter, using [iLOrest](http://hpe.com/info/resttool) version 3.5.
+
+The first instruction selects explicitly the `port.v1_3_1` data type that includes this adapter. Using a `ilorest select port.` data would also select other data versions (i.e. data types related to non RDE adapters). If several data types are select, the filtering of properties is harder.
+
+Then, it retrieves the `FlowControlConfiguration` of port 0, sets it to a different value. The last instruction gets again the value of this property to show that it has changed without any reboot.
+
+**NOTE**: No `commit` instruction was needed with this version of iLOrest to modify this property. This may be different with later versions.
+
+![Set port property in RDE network adapter](/img/setrdenetworkadapterproperty.png "Set port property in RDE network adapter")
+
+If you want to perform the same configuration using another Redfish client, you can send a PATCH request toward `{{iloURI}}/redfish/v1/Chassis/1/NetworkAdapters/DE081000/Ports/0` with the following body:
+
+```json
+{
+    "Ethernet": {
+        "FlowControlConfiguration": "TX"
+    }
+}
+```
+
+## Conclusion
+
+By leveraging several standards published before the Redfish standard (PLDM base specification, MCTP), the DMTF enhanced substantially the flexibility and ease of use of server management.
```

Comment:
swap the position of the words "enhanced" and "substantially".
You also might want to give a bit more of a plug to the HPE Developer blog in the conclusion with something like "Don't forget to check out some of my other blog posts on the HPE Developer portal to learn more about Redfish management." (and hotlink to the blog page)

Reply:
I've added some more meat in the conclusion.... Actually it was still in a "work in progress" state.